build.gradle

```diff
@@ -2,6 +2,9 @@
 
 buildscript {
     ext.kotlin_version = '1.4.21'
```

Comment:
If I'm not wrong, Lifecycle `2.3.0` is built against Kotlin `1.4.30`. What about we increase this version together? 🤔

Reply:
Yes!